### ISSUE TITLE
Possible bug

### ISSUE THREAD

**Issue by ericschrauth** [tabulapdf/tabula#928]
When viewing the preview, on lines that wrap in a cell it shows a space between the last word on one line and the first word on the next line.  But when you export CSV, the space is missing

**Comment by drmjp93** [comment]
Does this work with mojave?

**Comment by jeremybmerrill** [comment]
Oh that's interesting... Are words being improperly squished together? (I.e. if one line ends with "this" and the second line starts with "that", do you end up with "thisthat" in the output?)

I'm assuming this is "Lattice" mode, do you remember if that's right? (but if you don't recall, Eric, that's fine, don't worry about it.)

**Comment by ericschrauth** [comment]
Yes that is correct, it becomes thisthat.  Yes it is lattice mode. If you look at the json output, there is a \r in between the words so it knows there is a return there.  Just doesn't show up in the csv it seems.
    On Monday, October 1, 2018, 9:08:55 PM CDT, Jeremy B. Merrill <notifications@github.com> wrote:  
 
 
Oh that's interesting... Are words being improperly squished together? (I.e. if one line ends with "this" and the second line starts with "that", do you end up with "thisthat" in the output?)

I'm assuming this is "Lattice" mode, do you remember if that's right? (but if you don't recall, Eric, that's fine, don't worry about it.)

**Comment by jeremybmerrill** [comment]
Are you able to share the PDF? That'd help in tracking this down. Thanks for reporting this.

**Comment by ericschrauth** [comment]
Here is a sample file that causes the problem
[test.pdf](https://github.com/tabulapdf/tabula/files/2444006/test.pdf)

**Comment by jeremybmerrill** [comment]
@ericschrauth thanks!

**Comment by jeremybmerrill** [comment]
Hi @ericschrauth What program are you using to open the resulting CSV? When I exported a CSV from Tabula using the file you provided, it did properly contain the returns. I'm curious if whatever program in which you're viewing the CSV ignores the `\r` chars.
